src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -128,7 +128,7 @@
 	 * @param {*} [state]
```

Comment:
[nit] Can you please add a jsdoc type as well

Reply:
yup, will add that first then merge.